### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by figsoda**
###### Description of changes

Found some typos using [typos](https://github.com/crate-ci/typos) and manually fixed a bunch of typos 

sorry for the pings, feel free remove your review and unsubscribe to this thread if you don't want to review this

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [x] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
Only looked at 2 dozen or so, but looks good

File: nixos/modules/services/security/tor.nix [hunk 1]
```diff
@@ -146,7 +146,7 @@ let
     ]))];
     description = lib.mdDoc (descriptionGeneric optionName);
   };
-  optionBandwith = optionName: mkOption {
+  optionBandwidth = optionName: mkOption {
```
Comment (fricklerhandwerk) [inline nixos/modules/services/security/tor.nix:149]: That one probably needs an alias for backwards compatibility.

File: nixos/modules/services/web-apps/pgpkeyserver-lite.nix [hunk 2]
```diff
@@ -41,7 +41,7 @@ in
         defaultText = literalExpression "head config.${sksOpt.hkpAddress}";
         type = types.str;
         description = lib.mdDoc ''
-          Wich ip address the sks-keyserver is listening on.
+          Which ip address the sks-keyserver is listening on.
```
Comment (fricklerhandwerk) [inline nixos/modules/services/web-apps/pgpkeyserver-lite.nix:44]:
```suggestion
          Which IP address the sks-keyserver is listening on.
```

**Review by fricklerhandwerk — APPROVED** [review]
This is a great improvement of overall user experience! Thanks a lot.

File: nixos/modules/services/security/tor.nix (hunk 1 above)
Comment (figsoda) [reply nixos/modules/services/security/tor.nix:149]: I think this is a let binding that shouldn't break any backwards compatibility

**Pull request merged**